packages/wdio-cli/src/utils.ts

```diff
@@ -11,6 +11,10 @@ import type { Options, Capabilities, Services } from '@wdio/types'
 
 import { ReplCommandArguments, Questionnair, SupportedPackage, OnCompleteResult, ParsedAnswers } from './types'
 import { EXCLUSIVE_SERVICES, ANDROID_CONFIG, IOS_CONFIG, QUESTIONNAIRE } from './constants'
+import { ConfigParser } from '@wdio/config'
+import { DesiredCapabilities, MultiRemoteCapabilities, W3CCapabilities } from '@wdio/types/build/Capabilities'
```

Comment:
It is not recommended to import from the `build` folder. What happens when the file changes? Better do use the imports provided by the package, e.g.:

```ts
import { Capabilities } from '@wdio/types'
// and us it via e.g. `Capabilities.MultiRemoteCapabilities`
```